conbench/tests/api/test_benchmarks.py

```diff
@@ -936,6 +936,16 @@ def test_create_benchmark_context_missing(self, client):
         assert resp.status_code == 400, f"unexpected response: {resp.text}"
         assert '"context": ["Missing data for required field."]' in resp.text
 
+    def test_create_benchmark_name_missing(self, client):
+        self.authenticate(client)
+        payload = _fixtures.VALID_PAYLOAD.copy()
+        del payload["tags"]["name"]
+        resp = client.post(self.url, json=payload)
+        # TODO: https://github.com/conbench/conbench/issues/935
+        # This here just quickly checks that there is a failure at all,
+        # that 'name' is required.
+        assert resp.status_code == 500
```

Comment:
This quickly confirms existence of https://github.com/conbench/conbench/issues/935, and at least locks in the 'name is required!' condition for now. 

I don't expect this test code to live for long.